### DISCUSSION TITLE
milvus集群中新增miluvsdata-minio节点报错

### DISCUSSION THREAD

**Discussion by fengchen8556203** [milvus-io/milvus#34266] [Q&A and General discussion]
搭建的milvus集群，需要进行扩容，所以新增miluvsdata-minio节点但是一直无法启动报错这个：
Error: WARNING: Expected number of all hosts (4) to be remote +1 (4) (*errors.errorString)
       6: internal/logger/logger.go:258:logger.LogIf()
       5: cmd/peer-s3-client.go:312:cmd.newPeerS3Clients()
       4: cmd/peer-s3-client.go:76:cmd.NewS3PeerSys()
       3: cmd/erasure-server-pool.go:80:cmd.newErasureServerPools()
       2: cmd/server-main.go:785:cmd.newObjectLayer()
       1: cmd/server-main.go:611:cmd.serverMain()
能帮忙看下可能是什么问题导致的吗？

**Comment by yhmo** [comment]
不过这看起来不像是milvus的问题啊......

**Reply by xiaofan-luan** [reply]
might be related with https://stackoverflow.com/questions/71834747/distributed-minio-self-contained-example-fails-server-expects-storage-api-ve

but you need to tell more details about how did you scale minio
